misc/docker-ci/build

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# To build Docker images for CI and push the artifacts to Docker Hub.
+# https://hub.docker.com/r/h2oserver/h2o-ci
+#
+# usage: misc/docker-ci/build variants...
+#        variants might be ubuntu1604 ubuntu2004 (corresponding misc/docker-ci/Dockerfile.$variant)
+
+set -xe
+
+IMAGE_NAME=h2oserver/h2o-ci
+
+SCRIPT_DIR=$(realpath $(dirname $0))
+PROJECT_ROOT_DIR=$(realpath "${SCRIPT_DIR}/../..")
+
+# The build directory must be empty to avoid include files which are should not be published.
+BUILD_DIR="$SCRIPT_DIR/.empty"
+mkdir -p "$BUILD_DIR"
+trap 'rm -rf "${BUILD_DIR}"' 0
```

Comment:
How about creating an empty directory called `docker-root` (or something similar) and register it to the repository. Then, we can avoid the complexity of creating and removing the directory.

Reply:
Done! a7a8e9d6d8b2157ac9371e7973300325ad5315c5